src/Components/DetailedReview.jsx

```diff
@@ -36,7 +36,9 @@ if (isLoading) {
             <p>{currentReviewByID.review_body}</p>
             <p>Category :- {currentReviewByID.category}</p>
             <p>Posted at {currentReviewByID.created_at}</p>
-            <p>Current votes :- {currentReviewByID.votes}</p>
+            <p>Current votes :- {currentVotes}</p>
+            <button className="voteButton" disabled={buttonDisable} onClick={() => {addVotes(review_id, setCurrentVotes, setButtonDisable, setVoteCastText)}}>Vote for this review!</button><br />
```

Comment:
Please add your functionality for updating the votes in this component instead, please